src/entities/trade.test.ts

```diff
@@ -1160,6 +1192,50 @@ describe('Trade', () => {
   })
   // v3 sdk price impact tests
   describe('#priceImpact', () => {
+    describe('with FOT sell fees', () => {
```

Comment:
nit - technically we only support FOT that goes through v2-pools, not v3-pools. But I think it's fine to write unit test coverage against v4 routes, since `Trade` object is being used by v2, v3 and mixed routes, as a common object.

Reply:
good point, updated to v2 routes since it should match our real world use case :D